src/utils/format.test.ts

```diff
@@ -0,0 +1,18 @@
+import { humanFileSize } from './format';
+
+describe('Format Tests', () => {
+  describe('humanFileSize', () => {
+    it('string should return correct human-readable file size', () => {
+      expect(humanFileSize(1551859712)).toEqual('1.4 GiB');
```

Comment:
Maybe add a test case for when the number of bytes is less than 1000 or 1024 to test the `if branch`.